examples/coap-rd.c

```diff
@@ -51,7 +51,7 @@
 
 typedef struct rd_t {
   UT_hash_handle hh;      /**< hash handle (for internal use only) */
-  coap_key_t key;         /**< the actual key bytes for this resource */
+  str uri_path;           /**< the actual key for this resource */
```

Comment:
Is there a reason for using string instead of Hash? Seems somewhat unrelated to the other changes and has probably performance (negligible?) impact. And break existing APi for no apparent reason

Reply:
Again this is all in PR #133 .